integration/e2e/e2e_test.go

```diff
@@ -243,13 +247,22 @@ func makeThriftBatchWithSpanCount(n int) *thrift.Batch {
 			Flags:         0,
 			StartTime:     time.Now().Unix(),
 			Duration:      1,
-			Tags:          nil,
-			Logs:          nil,
+			Tags: []*thrift.Tag{
+				{
+					Key:  "x",
+					VStr: stringPointer("y"),
+				},
+			},
+			Logs: nil,
 		})
 	}
 	return &thrift.Batch{Spans: spans}
 }
 
+func stringPointer(s string) *string {
```

Comment:
this seems unnecessary. that has to be a better way to inline a string pointer than this

Reply:
I've updated to declare the variable first and then reference it in the struct.